deploy/helm-chart/harbor-operator/templates/instance/project.yaml

```
@@ -10,13 +10,15 @@ spec:
   parentInstance:
     name: {{ $instance.name }}
   name: {{ $project.name }}
+{{- if $project.storageLimit }}
+  storageLimit: {{ toYaml $project.storageLimit }}
```

I think the `toYaml` is not needed here because `$project.storageLimit` should be a simple integer...

```suggestion
  storageLimit: {{ $project.storageLimit }}
```

---

:+1: